### ISSUE TITLE
hanami-cli windows fork is unimplemented on this machine

### ISSUE THREAD

**Issue by dsisnero** [hanami/hanami#1458]
```
C:\windows_home\repos\github.com\dsisnero\hanami_mastery>bundle exec hanami assets compile
The `hanami` executable in the `hanami-cli_bulma` gem is being loaded, but it's also present in other gems (hanami-cli).
If you meant to run the executable for another gem, make sure you use a project specific binstub (`bundle binstub <gem_name>`).
If you plan to use multiple conflicting executables, generate binstubs for them and disambiguate their names.
C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/assets/command.rb:85:in `fork': fork() function is unimplemented on this machine (NotImplementedError)
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/assets/command.rb:85:in `fork_child_assets_command'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/assets/command.rb:61:in `block in call'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/assets/command.rb:61:in `map'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/assets/command.rb:61:in `call'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/command.rb:40:in `call'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/hanami-cli-2.2.0.beta2/lib/hanami/cli/commands/app/command.rb:40:in `call'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/dry-cli-1.2.0/lib/dry/cli.rb:117:in `perform_registry'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/gems/dry-cli-1.2.0/lib/dry/cli.rb:66:in `call'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/bundler/gems/hanami-cli_bulma-4730db89c8f0/exe/hanami:11:in `<top (required)>'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/bin/hanami:25:in `load'
        from C:/windows_home/ruby/ruby-3.3.4-1/lib/ruby/gems/3.3.0/bin/hanami:25:in `<main>'

```

**Comment by timriley** [comment]
Closing this as a dup of https://github.com/hanami/cli/issues/219. We have an open PR to address this (https://github.com/hanami/cli/pull/221) that I aim to have merged before the release of Hanami 2.2. Thanks for filing this anyway!

**Comment by pcopissa** [comment]
This fix does not work on a Windows platform. See https://github.com/hanami/cli/pull/221#issuecomment-2448705630 for a possible alternative solution.
